spec/factories/partnership.rb

```diff
@@ -16,6 +16,12 @@
     challenge_reason { "mistake" }
   end
 
+  trait :not_challenged do
+    challenged_at { nil }
+    challenge_reason { nil }
+    challenge_deadline { nil }
```

Comment:
This shouldn't have `challenge_deadline` in for "not challenged"

Reply:
The challenge_deadline is assigned randomly by the factory. We need to explicitly set it to nil.